view/frontend/web/js/teaser.js

```diff
@@ -54,56 +49,21 @@ define([
                     return;
                 }
                 this.reviewsData(JSON.parse(xhr.responseText));
-                this.populateReviewTabCount();
             }.bind(this));
             xhr.send();
         },
-
         getNumFullStars: function getFullStars() {
             //if the reviews are 4.8 or above return 5 else return the reviews
             return (this.reviewsData().avgRating >= 4.75) ? 5 : Math.round(this.reviewsData().avgRating);
         },
-
         hasHalfStar: function hasHalfStar() {
             let halfStarValue = (this.reviewsData().avgRating - this.getNumFullStars()).toFixed(2);
             return halfStarValue > 0.25 && halfStarValue <= .75;
         },
-
         getNumEmptyStars: function getNumEmptyStars() {
             return (5 - (this.getNumFullStars() + (this.hasHalfStar() ? 1 : 0)));
         },
 
-        writeReview: function writeReview() {
```

Comment:
@joshfedo where does "writeReview" end up going? I want to make sure it still works